modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/portlet_url/create_action_url.es.js

```diff
@@ -0,0 +1,5 @@
+export default function createActionURL(basePortletURL, parameters) {
+	parameters.p_p_lifecycle = '1';
```

Comment:
`parameters = {}`, it's an optional argument. Without it, code could break on this line.